### ISSUE TITLE
Tertiary Button in Dialog doesn't meet WCAG 2 requirements

### ISSUE THREAD

**Issue by maspychaj-sage** [Sage/carbon#4431]
### Current behaviour

Background color of Dialog causes color contrast issue when tertiary button is inside.

### Expected behaviour

Dialog background color doesn't causes contrast issues.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/condescending-grass-8wyif?file=/src/index.js:688-696

### JIRA Ticket (Sage Only)

SBS-20392

### Carbon Version

87.0.1

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Anything else we should know?

![image](https://user-images.githubusercontent.com/75613789/134891255-490e3375-7722-4689-afd1-89ca79b8547b.png)

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by mkrds** [comment]
@harpalsingh Could you check it out please?

**Comment by ingridzillinger** [comment]
Due to the new brand/rebranding, the background colour will change. Accessibility experts were involved, so this is a temporary problem.

**Comment by clairedenning** [comment]
From the DS team: Dialogs have been updated as part of the rebrand, and now have a white background. Contrast should no longer be an issue. The [zeroheight page](https://sage-ds.zeroheight.com/styleguide/s/54092/p/497e21-dialog/b/310ef2) has not been updated yet, but the figma link at the top of the page is up to date :)

**Comment by DlgSHi** [comment]
This will be fixed in the future
